introspection-engine/introspection-engine-tests/tests/re_introspection/mod.rs

```diff
@@ -1597,8 +1597,8 @@ async fn do_not_try_to_keep_custom_many_to_many_self_relation_names(api: &TestAp
     let final_dm = indoc! {r#"
         model User {
           id            Int         @id @default(autoincrement())
-          User_B        User[]      @relation("FollowRelation")
           User_A        User[]      @relation("FollowRelation")
+          User_B        User[]      @relation("FollowRelation")
```

Comment:
So this is a change in the expectation, because we're now sorting these?

Reply:
The sorting is there to avoid the ordering of fields changing even more. I didn't spend much time on it but wasn't able to replicate exactly what the old code was doing.